### PR TITLE
DES-8191: use GetLastInputInfo api

### PR DESCRIPTION
**Pull request by lneir**
greatly simplify logic by using win32 GetLastInputInfo api.

### CODE REVIEW

File: minuet/symphony/Symphony/Symphony/Plugins/ActivityPlugin.cs [hunk 1]
```diff
@@ -85,310 +64,37 @@ void onTimer(object state)
                 numOfSecondsWaited = 0;
                 activityOccurred = false;
 
+                Console.WriteLine("send at " + DateTime.Now);
```
Comment (mattharper) [inline minuet/symphony/Symphony/Symphony/Plugins/ActivityPlugin.cs:67]: Remove?

Comment (lneir) [inline minuet/symphony/Symphony/Symphony/Plugins/ActivityPlugin.cs:67]: just submitted another pr to remove log line
